mozetl/shield/privacy_prefs.py

```diff
@@ -23,6 +23,7 @@
     ("breakage", "payload/breakage", None, StringType()),
     ("notes", "payload/notes", None, StringType()),
     ("study", "payload/study", None, StringType()),
+    ("study_state", "payload/study_state", None, StringType()),
```

Comment:
You should add `study_state` to the failing test inputs. From the travis logs:

```
>       assert row_to_dict(actual) == create_row(result_payload)
E       AssertionError: assert {'branch': 't...problem', ...} == {'branch': 'th...problem', ...}
E         Omitting 7 identical items, use -vv to show
E         Left contains more items:
E         {'study_state': None}
E         Use -v to get the full diff
```

If you omit the field, it still gets filled in as a null value because it's included with the table schema.